integration/cmd/csv-to-json/main.go

```diff
@@ -0,0 +1,102 @@
+package main
+
+import (
+	"bytes"
+	"encoding/csv"
+	"encoding/json"
+	"flag"
+	"io/ioutil"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/golang/glog"
+)
+
+func main() {
+	flag.Parse()
+	ETL()
+}
+
+// ETL extracts result from csv and convert to json format
+//
+// Optimization for future: ETL could be a generic framework
+//
+// Extraction
+// Extraction part load data from sources which could has many kind of readers(json, csv, etc)
+// Extraction put loaded data into a channel
+//
+// Transformation
+// Transformation part read data from Extraction's result, doing things(user
+// defined functions) and output result to another channel
+// There might be trick logic in this step when input information is not separated
+// by lines:
+// Input string might be: last part for object 1 | first part for object 2
+// It need to implement buffer to handling this
+// Transformation part could have a pool of worker to execute
+//
+// Loader
```

Comment:
Why call it loader? Mostly it dumps contents, is it?

Reply:
Yes